src/Orchard/ContentManagement/DefaultContentManager.cs

```diff
@@ -95,7 +95,7 @@ public class DefaultContentManager : IContentManager {
             var context = new ActivatingContentContext {
                 ContentType = contentTypeDefinition.Name,
                 Definition = contentTypeDefinition,
-                Builder = new ContentItemBuilder(contentTypeDefinition)
+                Builder = new ContentItemBuilder(contentTypeDefinition, _contentDefinitionManager)

```

Comment:
I'd rather pass the partdefinition at this point. To be consistent with the first parameter.